### DISCUSSION TITLE
Best way to reorder tabs?

### DISCUSSION THREAD

**Discussion by tve** [mbnuqw/sidebery#1576] [Q&A] [answered]
I'm using sidebery with floorp and the lepton UI with auto-hide sidebar. One of the real annoyances is that it's tricky to reorder tabs 'cause the sidebar keeps wanting to close. Even if I go into the settings and turn the auto-close off the sidebar width isn't exactly generous for doing a major reordering of tabs (but I wouldn't want it wider for normal use). Ideal would be a separate full-page tab to do reordering, etc. Does that exist? Is there a recommended solution I'm missing?

**Comment by mbnuqw** [comment, marked as answer]
Addons can't control sidebar width, so until [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1408077) get resolved, there is nothing can be done from Sidebery's perspective. Currently, userChrome.css(lepton UI) and maybe Floorp styles are responsible for sidebar hiding, so I can only suggest asking there.
